spec/lib/card/content_spec.rb

```diff
@@ -89,7 +89,7 @@
     "\n        More urls: ",
     "<a class=\"external-link\" href=\"http://wagn.com/a/path/to.html\">wagn.com/a/path/to.html</a>",
     "\n        ",
-    "<a class=\"external-link\" href=\"http://localhost:2020/path?cgi=foo&bar=baz\">http://localhost:2020/path?cgi=foo&bar=baz</a>",
+    "<a class=\"external-link\" href=\"http://localhost:2020/path?cgi=foo+bar=baz\">http://localhost:2020/path?cgi=foo+bar=baz</a>",
```

Comment:
The point isn't the correctness of the URLs, but the handling of &.  I don't think it is a problem that they are encoded, and it may be correct that they get encoded for this context.  In other words, they may have been more broken the way it was.  Can you show that this is functionally wrong?